src/pages/Delivery/index.jsx

```diff
@@ -35,12 +42,11 @@ const Delivery = () => {
   if (hasError) {
     return <pre>Error</pre>
   }
-
   return (
     <Layout
       packageId={id}
-      headerTitle="En camino..."
-      headerSubtitle="Vamos con tu envio..."
+      headerTitle={headerStatus[currentStatus].headerTitle}
+      headerSubtitle={headerStatus[currentStatus].headerSubtitle}
```

Comment:
What happens in this status?

![10155.png](https://user-images.githubusercontent.com/66505715/142857203-02cf86b8-50c9-44ca-9183-83ac210c824e.png)

I think it would give an error because that property does not exist in the object

```Javascript
headerStatus['statusUncertain'].headerTitle // => Cannot find headerTitle in Undefined
```

Reply:
it won't throw an error because in this page this status should not be used

Reply:
And if is added the optional chaining operator `?` to avoid errors?

```javascript
objectExist['propertyDoesntExist']?.hola // => undefined
```

```suggestion
      headerTitle={headerStatus[currentStatus]?.headerTitle}
      headerSubtitle={headerStatus[currentStatus]?.headerSubtitle}
```